src/ugeneui/src/app_settings/user_apps_settings/UserApplicationsSettingsGUIController.cpp

```diff
@@ -111,7 +113,15 @@ UserApplicationsSettingsPageWidget::UserApplicationsSettingsPageWidget(UserAppli
         it.next();
         langCombo->addItem(it.key(), it.value());
     }
-    styleCombo->addItems(QStyleFactory::keys());
+    auto keys = QStyleFactory::keys();
+    for (const auto& key : keys) {
+        if (STYLE_KEYS_FIXED_REGISTER.keys().contains(key)) {
+            const QString& fixedName = STYLE_KEYS_FIXED_REGISTER.value(key);
+            keys.removeOne(key);
```

Comment:
You are changing the collection you currently iterate?

Reply:
See you don't like it. Fixed